docs/configuration-reference/components/velero.md

```diff
@@ -31,64 +31,64 @@ Velero component configuration example:
 # velero.lokocfg
```

Comment:
Can we please have the commit a bit more informative? Something like?

`docs: fix formatting for velero`